src/game/server/ddracechat.cpp

```diff
@@ -771,6 +771,14 @@ void CGameContext::ConUnPractice(IConsole::IResult *pResult, void *pUserData)
 		return;
 	}
 
+	char aBuf[256];
+	if(Teams.Count(Team) > g_Config.m_SvMaxTeamSize && pSelf->m_pController->Teams().TeamLocked(Team))
+	{
+		str_format(aBuf, sizeof(aBuf), "Can't disable practice. This team exceeds the maximum allowed size of %d players for regular team", g_Config.m_SvMaxTeamSize);
```

Comment:
```suggestion
pSelf->Console()->Print(
			IConsole::OUTPUT_LEVEL_STANDARD,
			"chatresp",
			"Can't disable practice. This team exceeds the maximum allowed size");
```

no need to specify the maximum teamsize here IMO

Reply:
Use `log_info` instead.

Reply:
It's sent to the player, `log_info` doesn't make sense here. The message is exactly the same as team0mode that specifies max size

Reply:
> `log_info` doesn't make sense here

It should behave exactly like `Print`. The `Print` function is deprecated and only a wrapper for `log_*` functions. And `OUTPUT_LEVEL_STANDARD` is translated to `LEVEL_INFO` which is `log_info`. New code should avoid using this function.